hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -1962,6 +1963,11 @@ public boolean waitForFlushes(long timeout) {
     }
   }
 
+  @Override
+  public Configuration getReadOnlyConfiguration() {
```

Comment:
Do we need return type to be ReadOnlyConfiguration to be very specific?
Configuration  also fine. Just asking...

Reply:
I think ReadOnlyConfiguration may be better. But I checked the code, the ReadOnlyConfiguration is only used in `CoprocessorEnvironment` and it use `Configuration getConfiguration()`, so here I use `Configuration` in the same way.